### PR TITLE
typos + added intro + conclusion

### PR DESCRIPTION
**Pull request by Pacane**

### CODE REVIEW

File: 1-jukito.md [hunk 1]
```diff
@@ -1,73 +1,58 @@
...
+# Introduction to Jukito
+In this second post of the Testing GWTP applications with Jukito, we present : Jukito!
 
-### What is Jukito?
-Jukito is a Java library that allows you to reduce the boilerplate
-when writing tests that cover code that is written using dependency
-injection. Typically, people use Mockito to mock their dependencies
-and it ends up being cumbersome.
-Jukito creates mocks for you for all the injected dependencies.
+The purpose of this post is to show the basics of Jukito, and what the advantages for you and for your project.
```
Comment (olafleur) [inline 1-jukito.md]: and TO SHOW what ARE the advantages ?

File: 1-jukito.md [hunk 2]
```diff
@@ -1,73 +1,58 @@
...
+* The annotation used to mock dependencies is `@Inject` instead of `@Mock`
+* The `initMocks()` function call that isn't needed anymore
 
-In the first case, it won't compile anymore, but in the second case,
-the tests will still pass.
+At first, it doesn't look like a big deal, but when the code base grows, things can get out of control. For example, imagine that we want to add a new dependency to the `Endpoint` class and inject it via constructor.
 
-Take note that code using Jukito must use Java's `@Inject` annotation
-to know that it has to inject a class its dependencies.
+With Mockito, this kind of change will produce a compilation error. On the other hand, with Jukito, everything still compiles fine, since the call site doesn't change.
```
Comment (olafleur) [inline 1-jukito.md:24]: the call site?

File: 1-jukito.md [hunk 3]
```diff
@@ -1,73 +1,58 @@
...
 
-### Ok, so what now?
-
-It also happens that Jukito leverages the power of Guice.
-As you probably know, Guice is a pretty powerful tool. So, Jukito has
-a lot of superpowers because of that.
+## OK, what now?
+Jukito uses [Google Guice](https://github.com/google/guice) and behind the scenes. However, even if you're not using Guice, you can still use Jukito.
```
Comment (olafleur) [inline 1-jukito.md]: C'est pas une phrase la première.

File: 1-jukito.md (hunk 2 above)
Comment (Pacane) [inline 1-jukito.md:24]: The place where a method is called a "call site".

Comment (olafleur) [inline 1-jukito.md:24]: is called is called ;)
Ok.

Comment (Pacane) [inline 1-jukito.md:24]: Yeah that :P

**Comment by olafleur** [comment]
LGTM

**Pull request merged**